modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml

```diff
@@ -241,6 +277,24 @@ components:
                     example: AccountGroup Name
                     type: string
             type: object
+        AccountGroupBrief:
+            properties:
+                description:
+                    example: AccountGroup Description
+                    type: string
```

Comment:
can you please remove also this? the briefs should contains just minimum information